### PR TITLE
Add support to `eth_blobBaseFee`, `eth_baseFee` and EIP4844 support to `eth_feeHistory`

### PR DESCRIPTION
**Pull request by Giulio2002**
Implemented:

https://www.quicknode.com/docs/hedera/eth_feeHistory
https://www.quicknode.com/docs/hedera/eth_baseFee
https://www.quicknode.com/docs/hedera/eth_blobBaseFee

### CODE REVIEW

**Comment by Giulio2002** [comment]
closes https://github.com/erigontech/erigon/issues/11263

File: turbo/jsonrpc/eth_system.go [hunk 1]
```diff
@@ -214,9 +218,69 @@ func (api *APIImpl) FeeHistory(ctx context.Context, blockCount rpc.DecimalOrHex,
...
+		return (*hexutil.Big)(common.Big0), nil
+	}
+	config, err := api.BaseAPI.chainConfig(ctx, tx)
+	if err != nil {
+		return nil, err
+	}
+	if config == nil {
+		return (*hexutil.Big)(common.Big0), nil
+	}
+	if !config.IsLondon(header.Number.Uint64() + 1) {
```
Comment (yperbasis) [inline turbo/jsonrpc/eth_system.go:278]: why +1 ?

Comment (Giulio2002) [reply turbo/jsonrpc/eth_system.go:278]: because baseFee endpoint applies to the next block compared to the head

**Review by yperbasis — APPROVED** [review]

**Comment by somnathb1** [comment]
@Giulio2002 Please back-port to `release/2.61`

**Pull request merged**